index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Apple</title>
+        <link rel="stylesheet" href="styles/reset.css">
+        <link rel="stylesheet" href="styles/style.css">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        
+        <style>
+            @import url('https://fonts.googleapis.com/css?family=Roboto:100');
+            </style>
+    </head>
+    <body> 
+        <div class="wrapp-all">
+            <header class="elements first-item">       
+                <ul>
+                    <li><img src="images/logo.png"></li>
+                    <li><span>Store</span></li>
+                    <li><span>Mac</span></li>
+                    <li><span>iPod</span></li>
+                    <li><span>iPhone</span></li>
+                    <li><span>iTunes</span></li>
+                    <li><span>iPad</span></li>
+                    <li><span>Support</span></li>
+                    <li><input type="search" name="search" placeholder="&#128269"></li>
+                  </ul>    
+                     
+            </header>
+            <main>                                                                                       
+               <h1>
+                   What will your verse be?
+                </h1>
+               
+                <h2>
+                   Everyone has something to share. See how peopleare using iPad<br>
+                   in unique ways to add to the world's story.
+                 </h2>
+               
+                <h3>
+                   Watch the "Your Verse" film and explore the stories
+                </h3>
+                    
+            </main>
+              
+
+        </div>
+       <br>   <br>   <br>   <br>   <br><br><br><br><br><br><br>
+        <aside>
```

Comment:
Move this aside tag and it's content into the footer, follow this guidelines for the footer
## Footer
* [ ] It should be wrapped by `<footer>` tags
* [ ] The top part is made of four `<a>` tags with `<div>` tags inside it
* [ ] The `<div>`s inside the `<a>` tags should use an image as a background
* [ ] The bottom part should include links like copyright, terms of use, and privacy policy